src/ssis_migrator/core/migration_planner.py

```
@@ -0,0 +1,98 @@
+                    "size": dtsx_file.stat().st_size,
+                    "complexity": "medium",  # TODO: Calculate actual complexity
+                    "estimated_effort": 2.0,  # hours
+                })
+            
+            plan = MigrationPlan(
+                packages=packages,
+                dependencies={},
+                estimated_duration=len(packages) * 2.0,  # hours
+                risks=["Complex transformations may require manual review"],
+                recommendations=["Start with simple packages", "Test thoroughly before production"]
+            )
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'estimated_duration' in constructor call

(E1123)